ui/src/app/components/results/test-case-result-details.component.ts

```diff
@@ -373,4 +382,40 @@ export class TestCaseResultDetailsComponent extends BaseComponent implements OnI
     this.hasSteps = hasSteps;
     console.log(hasSteps);
   }
+
+  openDryRun() {
+    this.matModal.open(DryRunFormComponent, {
+      height: "100vh",
+      width: '60%',
+      position: {top: '0px', right: '0px'},
+      panelClass: ['mat-dialog', 'rds-none'],
+      data: {
+        testCaseId: this.testCase.id
+      },
+    })
+  }
+
+  public fetchVersion(workspaceVersionId:number){
+    this.chromeRecorderService.isChromeBrowser();
+    this.chromeRecorderService.pingRecorder();
+    this.workspaceVersionService.show(workspaceVersionId).subscribe((res)=>{
+      this.version = res;
+      if (this.version.workspace.isWebMobile) {
+        this.chromeRecorderService.isChromeBrowser();
```

Comment:
what is the usecase for making call to `isChromeBrowser` & `pingRecorder` twice?

Reply:
Added by mistake removed